base/install_node

```diff
@@ -79,17 +79,22 @@ function verifyBinaryExists(version, cb) {
 
 
 function downloadAndInstallVersion(version, cb) {
-  var downloadUrl = 'https://storage.googleapis.com' +
-    '/gae_node_packages/node-' + version + '-linux-x64.tar.gz';
   var destPath = '/nodejs';
-  var cmd = 'curl ' + downloadUrl + '| tar xzf - -C ' + destPath +
-    ' --strip-components=1';
+  var baseUrl = 'https://storage.googleapis.com/gae_node_packages';
+  var cmd = '/opt/nodejs/verify_install_node ' + baseUrl +
+    ' ' + version + ' ' + version + '-SHASUMS256.txt.asc ' + destPath +
+    ' WARN_ON_ERROR';
```

Comment:
Reading this code, and connected the dots with the contents of `verify_install_node` it seems that we will download the SHASUMS from `https://storage.googleapis.com/gae_node_packages/v6.9.2-SHASUMS.txt.asc`. Is that correct? Are these files being mirrored into the GCS bucket at the moment?